resources/authgear/templates/en/web/authflowv2/__new_password_field.html

```diff
@@ -36,13 +36,12 @@
         `
       )
     }}
-    {{ $has_pw_input_err_msg := $has_new_pw_err }}
     {{ $pw_input_error_class := "input__error-message"}}
-    {{ if not $has_pw_input_err_msg }}
+    {{ if (eq $.NewPasswordErrorMessage "") }}
```

Comment:
`{{ if $.NewPasswordErrorMessage }}`

Reply:
Will update as `{{ if (not $.NewPasswordErrorMessage) }}`
Quoting [golang doc](https://pkg.go.dev/text/template#hdr-Actions) here for reference 🙏 

```
{{if pipeline}} T1 {{end}}
	If the value of the pipeline is empty, no output is generated;
	otherwise, T1 is executed. The empty values are false, 0, any
	nil pointer or interface value, and any array, slice, map, or
	string of length zero.
	Dot is unaffected.
```